website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -6350,6 +6350,11 @@ $ python -m apache_beam.runners.portability.expansion_service_test -p $PORT_FOR_
 
 Currently Python external transforms are limited to dependencies available in core Beam SDK Harness.
 
+#### 13.1.3. Creating cross-language Go transforms
+
+Go currently does not support creating cross-language transforms, only using cross-language
+transforms from other languages.
```

Comment:
Consider adding a JIRA task reference to this text for folks to track/contribute/refer to.

Reply:
Done.